plugins/parsers/csv/README.md

```diff
@@ -73,6 +73,9 @@ values.
   ## will be added as fields.
   csv_tag_columns = []
 
+  ## Set to true to let the column tags overwrite the metadata and default tags.
+  csv_tag_overwrite = false
```

Comment:
does this need to be specified in the `parsers/registry.go`?

Reply:
Not in the parsers registry, as it is a new feature and the code there is for backward compatibility for old implementations. Confirmed that with @srebhan, but I'm not sure now that I had a look at `config/config.go`..

Reply:
@srebhan can you confirm?

Reply:
@Hipska I went forth and back regarding the option and I'd say you probably should add new options for existing parsers in a backward compatible session. My reasoning is that if some external plugin uses the old way it would not be able to have the option otherwise. So my border-line would be "don't add new parser-plugins, but new options to old parsers". Does that make sense?

Reply:
It does definitely makes sense to not add new parser the old way. But also, that external plugin would not know/use this new option anyway? Or would it be able to use that automatically maybe? In that last case, yes it makes sense to make the new option old-style compatible..

Reply:
Well the user of an external plugin might see the option in the documentation and try to use it in the external plugin. That's the use-case. And I don't want us to explain why it works in telegraf not outside. So please just add it and we will remove that altogether in 2.0.

Reply:
Then that user can as well just convert his external plugin to new style? Or did you just implied that the external plugin would be able to use that option then automatically?

Reply:
There are users that just **use** existing external plugins... Please add the option. We have this old-style as part of the API contract, so please keep it. While I see an exception for totally new parsers, I think the effort is low enough to keep that contract for existing parsers.